src/WebInterface.hs

```diff
@@ -97,13 +98,13 @@ viewProjectInfo info =
   let
     owner = Project.owner info
     repo  = Project.repository info
+    ownerUrl = format "/{}" [owner]
     repoUrl  = format "/{}/{}" [owner, repo]
   in
     p $ do
-      a ! href (toValue repoUrl) $ do
-        toHtml owner
-        void "\x2009/\x2009" -- U+2009 is a thin space.
-        toHtml repo
+      a ! href (toValue ownerUrl) $ (toHtml owner)
+      void "\x2009/\x2009" -- U+2009 is a thin space.
+      a ! href (toValue repoUrl) $ (toHtml repo)
 
 -- Renders the body html for the index page.
 viewIndex :: [ProjectInfo] -> Html
```

Comment:
We can point the owner link in `viewProject` at the Hoff project page now, instead of at the GitHub page. I think that would make navigation a bit nicer.

Reply:
Sure!